src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -3858,16 +3858,16 @@ Activate by holding the key for the character you want to add an accent to, then
     <value>Back</value>
   </data>
   <data name="BugReportBtn.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
-    <value>Bug report</value>
+    <value>Create a bug report file</value>
   </data>
   <data name="BugReportTooltip.Text" xml:space="preserve">
-    <value>Bug report</value>
+    <value>Create a bug report file</value>
   </data>
   <data name="DocsBtn.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
-    <value>Documentation</value>
+    <value>Online documentation</value>
   </data>
   <data name="DocsTooltip.Text" xml:space="preserve">
-    <value>Documentation</value>
+    <value>Go to online documentation on MS Learn</value>
```

Comment:
Where is this.  OOBE has "Documentation on Microsoft Learn"

![image](https://github.com/microsoft/PowerToys/assets/1462282/ef7b6a74-be8b-4da2-a2eb-8f0985f1691c)

Reply:
Clint, it's a **tooltip**

Reply:
ok, on quick source code look up, looks like it is on update which is the update.

![image](https://github.com/microsoft/PowerToys/assets/1462282/707528ca-c51f-429b-8b71-01ccc029eca9)